From leann.ogasawara@canonical.com  Wed Jun 16 13:29:36 2010
From: Leann Ogasawara <leann.ogasawara@canonical.com>
Date: Thu, 10 Jun 2010 14:51:51 -0700
Subject: USB: option: Remove duplicate AMOI_VENDOR_ID
To: smurf@smurf.noris.de
Cc: linux-usb@vger.kernel.org
Message-ID: <1276206711.1221.3631.camel@emiko>


AMOI_VENDOR_ID is defined twice.  Remove the duplicate entry and move
the AMOI_PRODUCT_9508 definition to be grouped with the other AMOI
product definitions.

Originally-by: Ben Collins <ben.collins@ubuntu.com>
Signed-off-by: Leann Ogasawara <leann.ogasawara@canonical.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/option.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -206,6 +206,7 @@ static void option_instat_callback(struc
 #define AMOI_PRODUCT_H01			0x0800
 #define AMOI_PRODUCT_H01A			0x7002
 #define AMOI_PRODUCT_H02			0x0802
+#define AMOI_PRODUCT_9508			0x0800
 
 #define DELL_VENDOR_ID				0x413C
 
@@ -263,9 +264,6 @@ static void option_instat_callback(struc
 #define BANDRICH_PRODUCT_1011			0x1011
 #define BANDRICH_PRODUCT_1012			0x1012
 
-#define AMOI_VENDOR_ID			0x1614
-#define AMOI_PRODUCT_9508			0x0800
-
 #define QUALCOMM_VENDOR_ID			0x05C6
 
 #define CMOTECH_VENDOR_ID			0x16d8
